### PR TITLE
Add logic for moving tasks between sprints.

### PR DESCRIPTION
**Pull request by AndriyKovalets**

### CODE REVIEW

File: src/app/core/models/task/changeSprintForTask.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export class ChangeSprintForTask {
+    sprintId?: number;
+}
```
Comment (herakros) [inline src/app/core/models/task/changeSprintForTask.ts]: needed empty line

File: src/app/presentation/components/workspace-components/task-components/workspace-user-task-list/workspace-user-task-list.component.css [hunk 2]
```diff
@@ -261,3 +254,29 @@ mat-panel-title span {
...
+
+.card-info{
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+  cursor: grab;
+  font-size: 12px;
+  min-height: 90px;
+}
```
Comment (herakros) [inline src/app/presentation/components/workspace-components/task-components/workspace-user-task-list/workspace-user-task-list.component.css]: needed empty line

File: src/app/core/models/task/changeSprintForTask.ts (hunk 1 above)
Comment (FeloniukM) [inline src/app/core/models/task/changeSprintForTask.ts]: white space

File: src/app/presentation/components/workspace-components/task-components/workspace-user-task-list/workspace-user-task-list.component.css (hunk 2 above)
Comment (FeloniukM) [inline src/app/presentation/components/workspace-components/task-components/workspace-user-task-list/workspace-user-task-list.component.css]: white space

**Review by FeloniukM — CHANGES_REQUESTED** [review]

**Review by FeloniukM — APPROVED** [review]

**Review by Sergey988 — APPROVED** [review]

**Pull request merged**